src/redux/appReducer.js

```diff
@@ -0,0 +1,26 @@
+import * as buttonActions from '../pressButtonLogic';
```

Comment:
These are not actions (redux actions), it is a helper/service, so please rename it.